WordPress/WordPressIntents/ar.lproj/Sites.strings

```diff
@@ -13,5 +13,7 @@
     <string>تحديد موقع</string>
     <key>s4dJhx</key>
     <string>للتأكيد فقط، هل أردت "${site}"؟</string>
+    <key>tVvJ9c</key>
+    <string>تحديد هدف الموقع</string>
```

Comment:
I noticed that the email from Apple about missing localization for intent does not include `ar` anymore, which shows that this solved that issue, and that the remaining locales for which Apple complains about missing entry for "Site Intent" in `Sites.strings` are just the ones that are not yet ready in GlotPress (but will surely be later in the sprint), so 👌 

> Dear Developer,
> 
> We identified one or more issues with a recent delivery for your app, "WordPress – Website Builder" 19.6 (19.6.0.1). Your delivery was successful, but you may wish to correct the following issues in your next delivery:
> 
> ITMS-90626: Invalid Siri Support - Localized description for custom intent: 'SelectSite' not found for locale: cy
> …

The list of locales for which that error is stilll reported in the email is: `bg`, `cy`, `da`, `hr`, `hu`, `is`, `nb`, `nl`, `pl`, `pt`, `pt-br`, `sk`, `sq`, `th`. Which indeed seem to correspond to the ones that have not yet been updated in this PR (but should be later in the sprint when more translations come in GlotPress) 👍

Reply:
Also, once again (sadly no surprise), shows that translators don't really read our hints and comments associated with each key (and totally ignored our "You can use the English version verbatim without translation for this entry." instructions we left for this one)… 🤷  Eh. It won't be displayed to the end user anyway… but still sometimes I wonder why we bother 😒